app/views/govuk_publishing_components/components/docs/cards.yml

```diff
@@ -0,0 +1,139 @@
+name: Cards (experimental)
+description: A grid of cards that have a link and a description
+body: |
+  The component renders the cards as an unordered list element and visually presents it as a grid. It doesn't use the [GOV.UK Design System grid](https://design-system.service.gov.uk/styles/layout/#grid-system) but sets its own so that it can separately target desktop, tablet and mobile columns.
+
+  You can display the grid in three or two columns on desktop. Browsers that don't support CSS grid always display one column.
+
+  You must set a `href` for all the links.
+
+  This component is currently experimental. If you are using it, please feed back any research findings to the GOV.UK Explore team.
+accessibility_criteria: |
+  The component must:
+
+  -  use the correct heading level hierarchy eg. if the component heading uses a `<h2>`, the subheadings must use a `<h3>`
+
+  It is also good practise to include a heading above the list to tell users what the list contains
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      heading: Topics
+      items:
```

Comment:
A useful YAML thing is to name a section of the data: 
```yaml
data: &default-example-data
  items:
   - link:
        text: Benefits
        path: http://www.gov.uk
      description: Includes eligibility, appeals, tax credits and Universal Credit
```

and then you can reference that everywhere you want to use the `items` array:

```yaml
data:
  two_column_layout: true
  <<: *default-example-data
```

A real life example is the [link component docs][1].

[1]:https://github.com/alphagov/govuk_publishing_components/blob/019cd86bc63c985703c796ecb1b75133853d70a8/app/views/govuk_publishing_components/components/docs/list.yml